docs/tutorials/23.2-monorepo-rush.md

```diff
@@ -136,6 +136,75 @@ rush add -p react --dev
 After adding new packages you sometimes need to run `rush update --purge --full` to really update all cache files.
 :::
 
+## Add a Blazor Pilet
+
+Since blazor pilets need some more settings we'll cover them here in more depth. Let's start by adding a `blazorwasm` project. To do so run:
+
+```sh
+dotnet new blazorwasm -o .\packages\blazor-pilet
+```
+
+We should also add these two lines to the `.gitignore` file:
+
+```
+obj/
+bin/
+piral~/
+```
+
+We now have a standard `blazorwasm` project. To convert it into a pilet we need to go through the following steps (you can read more about this in the [Piral.Blazor README](https://github.com/smapiot/Piral.Blazor)):
+
+- Open `packages\blazor-pilet\blazor-pilet.csproj` and add the following inside the `PropertyGroup` section at the top:
+    ```xml
+    <PiralInstance>../app-shell/dist/emulator/app-shell-1.0.0.tgz</PiralInstance>
+    <Monorepo>enable</Monorepo>
+    ```
+- Install the `Piral.Blazor.Tools` and `Piral.Blazor.Utils` NuGet packages.
+- Rename *Program.cs* to *Module.cs*, and make sure to make the `Main` method an empty method.
+
+Now we need to prepare the Piral instance to be able to include blazor pilets with these steps:
+
+- Add the `blazor` dependency by running `rush add -p blazor` inside `packages\app-shell`
+- Add the `piral-blazor` dependency by running `rush add -p piral-blazor` inside `packages\app-shell`
+- In `packages\app-shell\src\index.tsx` add the import `import { createBlazorApi } from 'piral-blazor';`
+- In `packages\app-shell\src\index.tsx` add `plugins: [createBlazorApi()], ` inside the `renderInstance` call
+- You might want tu run `rush update --full --purge` just to be sure
+- Rebuild the app-shell via `rushx build`
+
+Now its time to build the pilet. Change into `packages\blazor-pilet` and run:
+
+```sh
+dotnet build
+```
+
+This will create JavaScript package for the blazor pilet in the `packages\piral~\blazor-pilet` folder. To make Rush aware of the new pilet we need to add it to the `projects` setting inside the *rush.json* file like this:
```

Comment:
```suggestion
This will create a JavaScript package for the blazor pilet in the `packages\piral~\blazor-pilet` folder. To make Rush aware of the new pilet, we need to add it to the `projects` setting inside the *rush.json* file like this:
```